pkgs/build-support/portable-service/default.nix

```diff
@@ -0,0 +1,75 @@
+{ pkgs, lib, stdenv }:
+/*
+  Create a systemd portable service image
+  https://systemd.io/PORTABLE_SERVICES/
+
+  Example:
+  …
```

Comment:
Was there supposed to be an example here?

Reply:
yes, I'll add a example once all is done